<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>_4fastmath</class>
 <widget class="QDialog" name="_4fastmath">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Быстрая математика</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(212, 218, 255);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/img/icons8-назад-50.png</normaloff>:/resource/img/icons8-назад-50.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>361</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="optionsLayout"/>
  </widget>
  <widget class="QLabel" name="feedbackLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 20pt &quot;STXihei&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="questionLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>381</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 18pt &quot;STXihei&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
